src/plugin/regions.js

```diff
@@ -341,6 +348,14 @@ class Region {
             this.wavesurfer.fireEvent('region-dblclick', this, e);
         });
 
+        this.element.addEventListener('contextmenu', e => {
+            if (preventContextMenu) {
+                e.preventDefault();
+            }
+            this.fireEvent('contextmenu', e);
+            this.wavesurfer.fireEvent('region-contextmenu', this, e);
```

Comment:
why fire 2 events here?

Reply:
This is in line with the other events (e.g. `dblclick`): one event is on the region itself and one is proxied by the main waveSurfer object.